app/javascript/components/enrollment/steps/Exposure.js

```diff
@@ -623,21 +623,21 @@ class Exposure extends React.Component {
                     <Form.Group as={Col} md="6" controlId="assigned_user" className="mb-2 pt-2">
                       <Form.Label className="nav-input-label">
                         ASSIGNED USER{schema?.fields?.assigned_user?._exclusive?.required && ' *'}
-                        <InfoTooltip tooltipTextKey="assignedUser" location="top"></InfoTooltip>
+                        <InfoTooltip tooltipTextKey="assigned_user" location="top"></InfoTooltip>
                       </Form.Label>
                       <Form.Control
                         isInvalid={this.state.errors['assigned_user']}
                         as="input"
-                        name="assignedUser"
-                        list="assignedUsers"
+                        id="assigned_user"
```

Comment:
@holmesie @kylietmo I've been talking to Sarah about this, but was wondering what you guys think about using name or id to refer to inputs (just want something consistent we can go with for the whole app)

![Screen Shot 2020-10-28 at 2 13 22 PM](https://user-images.githubusercontent.com/9956561/97480405-bb337f80-1929-11eb-9d3a-c63997923735.png)

Reply:
Sorry I haven't dived into this - so is the issue that setting `id` rather than `name` results in this warning?

Reply:
no worries, and to answer your question, yes. it seems like people on google are saying that this warning comes from using the same id for multiple inputs.

@sgober and I think this is strange though because although we use jurisdiction_path and assigned_user as id's in enrollment and on the patient page, they're on separate pages (not sure if these warnings are specifically referring to those fields but those are examples of fields where we are using the same id strings)

Reply:
I think I fixed this here: https://github.com/SaraAlert/SaraAlert/pull/450/commits/45e4d7764a1ddb395cd4d38108a1819ca1841c23